test/Makefile

```diff
@@ -39,9 +39,9 @@ alltests : test_runner
 	$(TEST_RUNNER) ${GTESTARGS}
 
 ping :
-	nc -vzw2 metadata.google.internal 80 >/dev/null 2>&1
+	nc -vzw2 169.254.169.254 80 >/dev/null 2>&1
 
 reset :
-	curl -Ss http://metadata.google.internal/reset >/dev/null 2>&1
+	curl -Ss http://169.254.169.254/reset >/dev/null 2>&1
```

Comment:
lol you shouldnt have changed this one, this is the test and it needs a fake metadata server. the real metadata server will never have a /reset endpoint :laughing: